dolphin-dnn/src/main/java/edu/snu/dolphin/dnn/layers/ConvolutionalLayer.java

```diff
@@ -77,13 +82,87 @@ public boolean isLearnable() {
   }
 
   /**
+   * Transform the given image to column form to facilitate matrix multiplication.
```

Comment:
This rearranges from the image into rows, doesn't it? If so, the method name `im2col` is misleading.
We can change `im2col` to `im2row`. However, it may not be familiar to people who use MATLAB or GNU Octave which provide an embedded `im2col` function.
Thus, we can change this function to transform from the image to columns and `col.mmul(getLayerParameter().getWeightParam()` to `getLayerParameter().getWeightParam().mmul(col)`. How do you think about this?